src/api-reference/direct-connects/hotel-service-4/v4.endpoints.md

```diff
@@ -407,7 +407,18 @@ POST /hotels/ratedetails
   ],
   "numGuests": 1,
   "guestCountryCode": "CA",
-  "searchSessionToken": "b41168ba-7ee1-4b68-9934-47f5c55337d6"
+  "searchSessionToken": "b41168ba-7ee1-4b68-9934-47f5c55337d6",
```

Comment:
It will be better to have two separate examples, first one as it was earlier (titled rate-details during shopping flow) and new example titled rate-details during modify/reshop flow)